cfme/tests/cloud/test_azure_manual.py

```diff
@@ -0,0 +1,241 @@
+import pytest
+
+pytestmark = [pytest.mark.manual]
+
+
+@pytest.mark.manual
+def test_vpc_env_selection():
+    """
+    Test selection of components in environment page of cloud instances
+    with and without selected virtual private cloud
+    Related to BZ 1315945
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: WebUI
+        initialEstimate: 1/2h
+        testSteps:
+            1. Provision an Azure Instance from an Image.
+            2. At the environment page, try to select components without vpc
+            3. At the environment page, try to select components without vpc with vpc
+        expectedResults:
+            1. Instance provisioned and added successfully
+            2. Items are selected successfully
+            3. Items are selected successfully
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_refresh_azure_provider_with_empty_ipv6_config_on_vm():
+    """
+    https://bugzilla.redhat.com/show_bug.cgi?id=1468700
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: Cloud
+        caseimportance: medium
+        initialEstimate: 1/4h
+        setup: prepare azure  with https://mojo.redhat.com/docs/DOC-1145084 or
+               https://docs.microsoft.com/en-us/azure/load-balancer/quickstart-create-basic-
+               load-balancer-cli
+        testSteps:
+            1. refresh azure provider
+        expectedResults:
+            1. no errors found in logs
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_vm_terminate_deletedisk_azure():
+    """
+    New for 5.6.1, when terminating a VM in Azure, we need to go to the
+    storage account and make sure the disk has also been removed.  You can
+    check the VM details for the exact disk location prior to deleting.
+    Note that Azure itself does not delete the disk when a VM is deleted,
+    so this may initially cause some confusion.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1353306
```

Comment:
Please use `Bugzilla:` header in docblock, or `pytest.mark.meta(automates=[BZ(number)])` so that this metadata information can be programatically accessed.